curated_data/specimen_source_map.csv

```diff
@@ -0,0 +1,492 @@
+SPECIMEN_SOURCE_NAME,CODE
```

Comment:
Specimen source values in Epic did not readily map to CDM values.  I've sent and email to DRNOC asking for suggestions.

In the mean time, I've left lots of null entries in the SPECIMEN_SOURCE_NAME column (viz. Epic data).  Not sure this is the smartest approach to this problem.  By way of alternatives, I could copy the CODE values over or drop all non-matching rows.